wormchain/x/wormhole/client/cli/genesis.go

```diff
@@ -406,10 +407,10 @@ func CmdGenerateGuardianSetUpdatea() *cobra.Command {
 	return cmd
 }
 
-func CmdSignAddress() *cobra.Command {
+func CmdTestSignAddress() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "sign-address",
-		Short: "sign the validator address to use for registering as a guardian.  read guardian key as hex in $GUARDIAN_KEY env variable. use --from to indicate address to sign.",
+		Use:   "test-sign-address",
+		Short: "Test method sign the validator address to use for registering as a guardian.  Use guardiand for produciton, not this method.  Read guardian key as hex in $GUARDIAN_KEY env variable. use --from to indicate address to sign.",
```

Comment:
<i class='graphite__hidden'>[Re: lines 413 to 413]<br/></i>

"produciton" -> "production"<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/1963?utm_source=unchanged-line-comment">Graphite</a>.</span>